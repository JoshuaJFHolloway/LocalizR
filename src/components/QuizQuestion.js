import React from 'react';

const QuizQuestion = () => {
  return (
    <p>This is our first Spanish question?</p>
  )
};

export default QuizQuestion;
