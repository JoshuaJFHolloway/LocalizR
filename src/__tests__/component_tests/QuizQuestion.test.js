import React from 'react';
import { shallow } from 'enzyme';
import QuizQuestion from '../../components/QuizQuestion';

describe('QuizQuestion', () => {
  let quizquestion = shallow(<QuizQuestion />);

  it('renders correctly', () => {
    expect(quizquestion).toMatchSnapshot();
  })
})
