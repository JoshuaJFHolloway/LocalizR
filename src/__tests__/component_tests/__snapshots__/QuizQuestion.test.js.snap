// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuizQuestion renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <QuizQuestion />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "This is our first Spanish question?",
    },
    "ref": null,
    "rendered": "This is our first Spanish question?",
    "type": "p",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "This is our first Spanish question?",
      },
      "ref": null,
      "rendered": "This is our first Spanish question?",
      "type": "p",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
